src/snipe/api/multisig_reference_QC.py

```
@@ -766,145 +987,167 @@
+        """
+
         if predict_extra_folds and genome_stats["Genome coverage index"] > 0.01:
-            predicted_fold_coverage = {}
-            predicted_fold_delta_coverage = {}
-            predicted_unique_hashes = {}
-            predicted_delta_unique_hashes = {}
-            nparts = 30
-            roi_reference_sig = self.reference_sig
+            FRACMINHASH_PRECISION = 1.00
+            if sample_sig.valid_kmers is not None and sample_sig.valid_kmers > 0:
+                FRACMINHASH_PRECISION = (sample_sig.total_abundance * sample_sig.scale) / sample_sig.valid_kmers
```

Ruff (0.8.2)

Local variable `FRACMINHASH_PRECISION` is assigned to but never used

Remove assignment to unused variable `FRACMINHASH_PRECISION`

(F841)